### ISSUE TITLE
About input c

### ISSUE THREAD

**Issue by conchincradle** [autonomousvision/stylegan_xl#26]
Sorry to bother you with naive questions, but I don't figure out it after two days (stylegan2 ada is also not clear), I will really appreciate it if you could reply to me!
I wrote like below to input a image with label into the Discriminator of your pretrained model
```
import torch
import dnnlib
import pickle
import legacy
import PIL
import numpy as np
network_pkl ='./pretrained/imagenet64.pkl'
device = torch.device('cuda')
with dnnlib.util.open_url(network_pkl) as f:
        D = legacy.load_network_pkl(f)['D']
        D = D.eval().requires_grad_(False).to(device)
image =torch.from_numpy(np.array(PIL.Image.open( './data/01.JPEG'))).float().permute(2,0,1)
image = image.reshape((1,3,64,64))
image = image.to(device)

c = torch.randn([1,1000,3,3],dtype=torch.float).cuda()
print(D(image,c))

```
But I still don't know how to write the label , could I ask if you could give an example of input c, (suppose here is class 0)?
Thank you very much!

**Comment by woctezuma** [comment]
Maybe from [the README](https://github.com/autonomousvision/stylegan_xl#generating-samples--interpolations):

> For class-conditional models, you can pass the class index via --class.
> A index-to-label dictionary for Imagenet can be found [here](https://github.com/xl-sr/stylegan_xl_release/blob/main/media/imagenet_idx2labels.txt).

**Comment by xl-sr** [comment]
see above :slightly_smiling_face:
